### ISSUE TITLE
WC: Command not found

### ISSUE THREAD

**Issue by danXNU** [BishopFox/bfinject#20]
When i executed `bash bfinject -P Reddit -L test`, i received this error:

`bfinject: line 43: wc: command not found`
`[!] "Reddit" was not uniquely found, please check your criteria.`

Where do i take the wc command from? I tried searching in Cydia but it's not in the default sources.

[Electra 1.0.1] - iOS 11.1.2

Thanks!

**Comment by riley-williams** [comment]
wc is in the Core Utilities package

**Comment by danXNU** [comment]
Thanks!!!

**Comment by othomas1984** [comment]
Core Utilities package was installed already when I launched Cydia for the first time, but does not seem to include wc (or many other handy tools such as any text editor, md5 utilities, etc.). Any idea why this might be?

I'm on 11.1.2. I did have LiberIOS installed first, and then installed the latest Electra with Tweaks, then rebooted and ran Electra without Tweaks. Could this be the problem by any chance? 

I've tried so many things to be able to run cycript and nothing's worked so far. Would love a little help.

Edit: I may have spoken too soon... I didn't realize there was a difference between `Core Utilities (/bin)` and just plain old `Core Utilities`. And for some reason when I first searched in Cydia on my device it didn't show `Core Utilities` as an option. It's now shown up though and I've installed it. So I'm going to give this another shot!!
